DOC: Suggest replacement for tostring_rgb (#25502)
matplotlib/matplotlib: doc/api/next_api_changes/deprecations/XXXXX-AL.rst

``RendererAgg.tostring_rgb`` and ``FigureCanvasAgg.tostring_rgb``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> These have no direct replacement. Consider using ``tostring_argb`` instead,

Reviewer: ```suggestion
... are deprecated with no direct replacement. Consider using ``tostring_argb`` instead,
```
